### PR TITLE
completed express-notes-api challenge

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
https://user-images.githubusercontent.com/103273265/177466722-3670f548-d6f2-4a54-93be-2ea6dfcb2f28.mov

https://user-images.githubusercontent.com/103273265/177466734-af2feefb-0f53-464a-9c07-b01efa29148e.mov

https://user-images.githubusercontent.com/103273265/177466739-da99f751-49cb-4822-a5f0-c4c9275f0ce0.mov

https://user-images.githubusercontent.com/103273265/177466742-6ead8f38-d528-4dc7-83e6-8217f8ea5c66.mov

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, good, just some minor tweaks.

**Comment by bryanvaldovinos** [comment]
https://user-images.githubusercontent.com/103273265/177612378-2f6a8831-1c87-464c-b7f7-30ce6a1fccc1.mov

File: express-notes-api/index.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        res.sendStatus(204);
+      }
+    });
+  }
+});
+
+app.put('/api/notes/:id', (req, res) => {
+  const id = Number(req.params.id);
+  const inputC = Object.keys(req.body)[0];
+  if (inputC === undefined) {
```
Comment (thebearingedge) [inline express-notes-api/index.js:98]: This one needs to be updated too.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-notes-api/index.js (hunk 1 above)
Comment (bryanvaldovinos) [reply express-notes-api/index.js:98]: got you

**Review by thebearingedge — APPROVED** [review]

**Comment by bryanvaldovinos** [comment]
https://user-images.githubusercontent.com/103273265/177657167-cc39507b-df3f-4c9e-a62c-b8f4ea283b0f.mov

**Pull request merged**